{
	"id": "CVE-2015-3220",
	"vulnerabilities": [
		{
			"id": "CVE-2015-3220",
			"description": "The tlslite library before 0.4.9 for Python allows remote attackers to trigger a denial of service (runtime exception and process crash).",
			"references": [
				{
					"source": "redhat",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1254215"
				},
				{
					"source": "redhat",
					"url": "https://github.com/trevp/tlslite/commit/aca8d4f898b436ff6754e1a9ab96cae976c8a853"
				},
				{
					"source": "redhat",
					"url": "https://groups.google.com/forum/#%21topic/tlslite-dev/MoWE7B0A4iU"
				}
			]
		}
	],
	"data_source": "mitre-v5"
}
